src/App.js

```diff
@@ -14,6 +14,17 @@ function App() {
     const [favouriteItems, setFavouriteItems] = useState([]);
     const [basketItems, setBasketItems] = useState([]);
 
+    useEffect(() => {
+        let basketProducts = JSON.parse(
+            window.localStorage.getItem('basketProduct')
```

Comment:
You don't need to access the `window` object. You can just call `localStorage.getItem`.

Reply:
I changed it